webapp/crashstats/crashstats/jinja2/crashstats/bug_comment.txt

```diff
@@ -6,7 +6,7 @@ Crash report: {{ full_url(request, "crashstats:report_index", crash_id=uuid ) }}
 {% if java_stack_trace %}
 Java stack trace:
 ```
-{{ java_stack_trace|safe }}
+{{ java_stack_trace|truncate(5000, True)|safe }}
```

Comment:
We have to truncate it here so we don't mess up the markdown formatting. Also, if we ever add additional bits after the stack trace, we don't want them truncated by the view.